.github/workflows/check-pull-request.yml

```diff
@@ -65,18 +65,6 @@ jobs:
     name: Test-Pallets
     runs-on: ubuntu-latest
     continue-on-error: false
-    strategy:
-      matrix:
-        pallet:
-          - pallet-ajuna-awesome-avatars
-          - pallet-ajuna-awesome-avatars-benchmarking
-          - pallet-ajuna-battle-mogs
-          - pallet-ajuna-nft-staking
-          - pallet-ajuna-nft-staking-benchmarking
-          - pallet-ajuna-nft-transfer
-          - pallet-ajuna-affiliates
-          - pallet-ajuna-tournament
-          - pallet-ajuna-wildcard
```

Comment:
I was just lazy in the beginning, but then I thought, why do we have it like that? With this pattern we have to add every crate in the workspace, not only the pallets. I think it is very easy to accidentally miss a crate, and then not test it. Hence, I would suggest running the tests on the workspace if you have nothing against that.

Reply:
I agree that it makes more sense to do it like this